packages/redux/src/addresses/actions/factories/fetchAddressesFactory.ts

```diff
@@ -37,11 +38,10 @@ const fetchAddressesFactory =
   (getAddresses: GetAddresses) =>
   (userId: User['id'], config?: Config) =>
   async (dispatch: Dispatch<FetchAddressesAction>): Promise<Address[]> => {
-    dispatch({
-      type: FETCH_ADDRESSES_REQUEST,
-    });
-
     try {
+      dispatch({
```

Comment:
Just noticing now, why move this to the `try` block? Shouldn't this happen anyway, regardless of what comes next? 🤔 

I mean, even for readability, the "start request" is independent, isn't it? (just trying to understand the rationale behind this change

Reply:
And after seeing some more files, this happens only in some places, not everywhere 😖

Reply:
To easily see the changes, I've created a new commit, to apply the same pattern to every action.

The point of setting the try for every code inside action it's to deal with possible exceptions in action, not only caused by the request itself. In this way, every error obtained when some action is called, will be stored in the state.